src/ckb-daemon/usb.h

```diff
@@ -39,6 +39,7 @@
 #define P_K95_PLATINUM  0x1b2d
 #define P_K95_PLATINUM_STR "1b2d"
 #define IS_K95(kb)      ((kb)->vendor == V_CORSAIR && ((kb)->product == P_K95 || (kb)->product == P_K95_NRGB || (kb)->product == P_K95_PLATINUM))
+#define IS_K95P(kb)     ((kb)->vendor == V_CORSAIR && ((kb)->product == P_K95_PLATINUM))
```

Comment:
shall i make it IS_PLATINUM or IS_K95_PLATINUM?
I can do this before commiting my work again and opening a new pull request